services/inc/rgbled.h

```diff
@@ -42,20 +43,6 @@ typedef enum
 #define LED_RGB           LED3_LED4_LED2
 #define LED_USER          LED1
 
-// FIXME: These should be pulled from platform_config.h
```

Comment:
The unit tests seem to be broken. Let's leave these `ifndef`s here with default values in addition to `platform_config.h` and create an empty `platform_config.h` for GCC platform in `platform/MCU/gcc/inc`.

Reply:
Fixed